lib/restful/jsonapi/restify_param.rb

```diff
@@ -43,7 +43,7 @@ def restify_belongs_to(relationship_name, relationship_data)
           relationship_key = relationship_name.to_s.underscore+"_attributes"
           {relationship_key => restify_data(relationship_name,relationship_data[:data])}
         else
-          if relationship_data[:data].nil?
+          if relationship_data[:data].nil? || relationship_data[:data].empty?
```

Comment:
`if relationship_data[:data].blank?` would remove the need for an `||`.

Reply:
Great point!